quick-start/src/test/java/com/marklogic/quickstart/integrationtests/IntegrationUtils.java

```diff
@@ -0,0 +1,292 @@
+			CodeFormat codeFormat, DataFormat dataFormat, Map<String, Object> options) 
+					throws IOException {
+		
+		String optionString = toJsonString(options).replace("\"", "\\\"");
+		
+		String optionsJson = "{ " + "\"mlcpPath\":\"" + mlcpPath + "\"," + "\"mlcpOptions\":" +
+                "{" +
+                    "\"input_file_path\":\"" + inputPath.replace("\\", "\\\\\\\\") + "\"," +
+                    "\"input_file_type\":\"\\\"documents\\\"\"," +
+                    "\"output_collections\":\"\\\"" + entityName + "\\\"\"," +
+                    "\"output_permissions\":\"\\\"rest-reader,read,rest-writer,update\\\"\"," +
+                    "\"output_uri_replace\":\"\\\"" + basePath.replace("\\", "/")
+                    .replaceAll("^([A-Za-z]):", "/$1:") + ",''\\\"\"," +
+                    "\"document_type\":\"\\\"" + dataFormat.toString() + "\\\"\",";
+        if (codeFormat.equals(CodeFormat.JAVASCRIPT)) {
+                optionsJson += "\"transform_module\":\"\\\"/MarkLogic/data-hub-framework/transforms/"
+                		+ "mlcp-flow-transform.sjs\\\"\"," + "\"transform_function\":\"transform\",";
+        } else {
+                optionsJson += "\"transform_module\":\"\\\"/MarkLogic/data-hub-framework/transforms/"
+                		+ "mlcp-flow-transform.xqy\\\"\"," + "\"transform_namespace\":\"\\\"http://marklogic.com/data-hub/"
+                		+ "mlcp-flow-transform\\\"\",";
+        }
+        optionsJson += "\"transform_param\":\"entity-name=" + entityName + ",flow-name=" + flowName + ",options=" 
+            			+ optionString + "\"" + "}}";
+        
+        JsonNode ipFlowJsonNode = new ObjectMapper().readTree(optionsJson);
+        
+		return ipFlowJsonNode;
+	}
+	
+	public String toJsonString(Object value) {
+        try {
+            return new ObjectMapper().writeValueAsString(value);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+    }
+	
+	public String getFlowName() {
+		return this.flowName;
+	}
+	
+	public void waitForReloadModules() {
+		try {
+			System.out.println("Waiting to deploy Modules");
+			Thread.sleep(10000);
+		} catch (InterruptedException e) {
+			e.printStackTrace();
+		}
+	}
+	
+	private void buildCookie() {
+		requestCookie = new Cookie.Builder("JSESSIONID", sessionID).setSecured(true)
+			      .setComment("session id cookie").build();
+	}
+	
+	private void createLoginCredentials(Response projectInitResponse) {
+		JsonPath projectInitJson = projectInitResponse.jsonPath();
+		loginInfo.projectId = projectInitJson.getInt("id");
+		loginInfo.environment = projectInitJson.getList("environments").get(0).toString();
+		loginInfo.username = "admin";
+		loginInfo.password = "admin";
+	}
+	
+	private FlowModel createFlowModel(String entityName, String flowType, DataFormat dataFormat, 
+			CodeFormat codeFormat, boolean useEsModel, String mappingName) {
+		flowModel = new FlowModel();
```

Comment:
really weird to use an instance variable here.  Why  not just local?

Reply:
Changed a few other instance variables to local variables where instance variables are not required.